src/firmware_new/boards/frankie_v1/lwip.h

```diff
@@ -40,7 +40,8 @@ extern "C"
 #include "lwip/tcpip.h"
 #endif /* WITH_RTOS */
 
-    /* USER CODE BEGIN 0 */
+/* USER CODE BEGIN 0 */
+#define SYS_LIGHTWEIGHT_PROT 1
```

Comment:
What's up with this?

Reply:
not sure 😢 its not user code, thats a setting ive toggled through cube, ill take a closer look and update this comment

Reply:
ok i remembered now, it doesn't generate this option when we select it on cube, so we have to do it ourselves

there is no `USER CODE` section in `lwipopts.h` so this was the next best place I could put it